raiden-dapp/src/components/navigation/Transfer.vue

```diff
@@ -58,37 +77,54 @@
         novalidate
         @submit.prevent="navigateToTransferSteps(target, amount)"
       >
-        <v-row class="transfer__form-container__title" no-gutters>
-          {{ $t('transfer.transfer-title') }}
-        </v-row>
         <div class="transfer__form-container__form">
-          <v-row class="transfer__form-container__form__inputs" no-gutters>
-            <v-col>
-              <address-input
-                v-model="target"
-                class="transfer__form-container__form__inputs__address-input"
-                :exclude="[token.address, defaultAccount]"
-                :block="blockedHubs"
-              />
+          <v-row class="transfer__form-container__form__heading" no-gutters>
+            <v-col
+              class="transfer__form-container__form__heading--title"
+              cols="2"
+            >
+              {{ $t('transfer.transfer-title') }}
             </v-col>
-            <v-col>
-              <amount-input
-                v-model="amount"
-                class="transfer__form-container__form__inputs__amount-input"
-                :token="token"
-                :placeholder="$t('transfer.amount-placeholder')"
-                :max="capacity"
-                limit
-              />
+            <v-col
+              class="transfer__form-container__form__heading--error"
+              cols="10"
+            >
+              <span>
+                {{ addressError }}
```

Comment:
Could you make sure this is never empty? Maybe make it a block element? I just want to avoid that this makes the components below jump up and down depending of if there is an error message or not.f

Reply:
If I leave an empty space for both messages it will not look according to the design. I see two solutions to this.
1. Find a shorter copy so all errors fit on one line
2. Rework the design to accommodate more extensive error messages

For now, perhaps we can assume that a user addresses the first error before messing up the second field :D

Reply:
Hmm. Okay. Let's say it is good enough for now. So maybe schedule it for a later UI improvement.